### PR TITLE
Update form element helper text

### PR DESCRIPTION
**Pull request by JCook21**
This pr works in conjunction with changes in modeler and small fixes in vue-form-elements to update helper text in the project.

### CODE REVIEW

File: resources/js/processes/modeler/components/inspector/TaskDueIn.vue [hunk 1]
```diff
@@ -1,16 +1,14 @@
 <template>
-    <div>
```
Comment (JCook21) [inline resources/js/processes/modeler/components/inspector/TaskDueIn.vue]: This extra div was causing alignment issues and removing it took care of these.

File: resources/lang/en.json [hunk 2]
```diff
@@ -972,16 +979,13 @@
   "Warnings": "Warnings",
   "no warnings to report": "No Warnings To Report",
   "Assignment Rules": "Assignment Rules",
-  "Enable Interstitial": "Enable Interstitial",
   "redirected to my next assigned task": "redirected to my next assigned task",
```
Comment (lisa-fehr) [inline resources/lang/en.json]: This was replaced with: Directs Task assignee to the next assigned Task
```suggestion
  "Directs Task assignee to the next assigned Task": "Directs Task assignee to the next assigned Task",
```

File: resources/lang/en.json [hunk 3]
```diff
@@ -682,11 +688,12 @@
   "success": "success",
   "Successfully imported": "Successfully imported",
   "Successfully saved": "Successfully saved",
-  "Summary screen that will be displayed when process finish with this End event.": "Summary screen that will be displayed when process finish with this End event.",
```
Comment (lisa-fehr) [inline resources/lang/en.json]: This was replaced with "Select Display-type Screen to show the summary of this Request when it completes" ✅  and "Select Screen to display this Task" ❌

File: resources/lang/en.json [hunk 4]
```diff
@@ -638,9 +641,12 @@
   "Select allowed users": "Select allowed users",
   "Select direction of gateway": "Select direction of gateway",
   "Select Direction": "Select Direction",
+  "Select Display-type Screen to show the summary of this Request when it completes": "Select Display-type Screen to show the summary of this Request when it completes",
   "select file": "select file",
   "Select List": "Select List",
   "Select group or type here to search groups": "Select group or type here to search groups",
```
Comment (lisa-fehr) [inline resources/lang/en.json]:
```suggestion
  "Select group or type here to search groups": "Select group or type here to search groups",
  "Select Screen to display this Task": "Select Screen to display this Task",
```

File: resources/js/processes/modeler/components/inspector/TaskAssignment.vue [hunk 5]
```diff
@@ -36,19 +37,20 @@
...
+            :helper="$t('Allows the Task assignee to reassign this Task')"
             :checked="allowReassignmentGetter"
             @change="allowReassignmentSetter">
         </form-checkbox>
 
         <div class="form-group">
 
             <div class="form-group special-assignment-header">
-                <label>{{ $t("Assign by Expression") }}</label>
+                <label>{{ $t("Assign by Expression: Use a rule to assign this Task conditionally") }}</label>
```
Comment (lisa-fehr) [inline resources/js/processes/modeler/components/inspector/TaskAssignment.vue]: Wondering if this is why it says:
"Missing Translation:  Use a rule to assign this Task conditionally"

I don't see "Use a rule to assign this Task conditionally" used anywhere on its own here.

**Pull request merged**